apps/client/src/components/Conclusion.js

```diff
@@ -78,42 +69,24 @@ const Conclusion = ({ checker, matomoTrackEvent }) => {
   return (
     <>
       <Paragraph>
-        Op basis van uw antwoorden vindt u hieronder wat voor uw activiteit van
-        toepassing is.
+        U bent klaar met de vergunningcheck. Dit is de uitkomst:
       </Paragraph>
 
-      {displayConclusions.map(({ title, description }) => (
-        <Fragment key={title}>
-          <Heading forwardedAs="h2">{title}</Heading>
-          <Markdown eventLocation={sections.CONCLUSION} source={description} />
-        </Fragment>
-      ))}
-
-      <HideForPrint>
-        {needsPermit && !contactConclusion && (
-          <ComponentWrapper marginBottom={40}>
-            <Button
-              type="button"
-              color="secondary"
-              onClick={handlePermitButton}
-            >
-              Vergunning aanvragen
-            </Button>
-          </ComponentWrapper>
-        )}
-
-        {!isIE && !isMobile && (
-          <ComponentWrapper>
-            <PrintButton
-              color="primary"
-              onClick={handlePrintButton}
-              type="button"
-            >
-              Conclusie opslaan
-            </PrintButton>
-          </ComponentWrapper>
-        )}
-      </HideForPrint>
+      {needsPermit && !contactConclusion && (
+        <NeedPermit
```

Comment:
I find these three components to have inconsistent names: NeedPermit, PermitFree, ContactConclusion

Also, the content of these files is pretty similar and the props you're passing is similar. They all have this flow:
- Title
- Optional text (Markdown or Paragraph)
- Buttons
- Optional text

I think it would be better to make one variable called `conclusionType` which can only be 'needPermit', 'needContact' and 'permitFree'. For each of these `conclusionType` make a configurable object with a title, optional texts (Components) and buttons. Then you can build the content of this page dynamically instead of having a lot of duplicate code.

What do you guys think? @afjlambert @svenjens

Reply:
Something similar to https://github.com/Amsterdam/vergunningcheck/pull/475/files ?